.github/actions/abort-release-candidate-v1/action.yml

```diff
@@ -0,0 +1,108 @@
+name: abort-release-candidate
+description: 'A GitHub Action to abort a release candidate'
+
+inputs:
+  token:
+    description: 'Token used for the GH CLI to close and move issue(s) to the correct column. See README for required permissions'
+    required: true
+  base:
+    description: 'The branch the release candidate will be merged into'
+    required: true
+  docs-repo:
+    description: 'The docs repository where the issue for the release candidate is located'
+    default: ''
+    required: false
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
+        # Checkout the release branch so we can get the release candidate version and reset it to the base branch later
+        ref: 'release'
+
+    - name: Checking out release branch and get release candidate version
+      shell: bash
+      id: get-release-candidate-version
+      run: |
+        # Get the version number from the prepare_release shell script and set it as an output
```

Comment:
Is this a standard shell script or a leftover of an old pattern we were doing?

Reply:
We use this in multiple places - it probably makes sense to move it to own shell script action instead of having this block copy pasted multiple times in different actions. 

https://github.com/dequelabs/axe-api-team-public/issues/115